api/migrations/schema/Version20220830180349.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220830180349 extends AbstractMigration {
+    public function getDescription(): string {
+        return '';
+    }
+
+    public function up(Schema $schema): void {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE TABLE o_auth_state (id VARCHAR(16) NOT NULL, state VARCHAR(255) NOT NULL, expireTime TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, createTime TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, updateTime TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, PRIMARY KEY(id))');
```

Comment:
Do we need 255 for the state? Isn't `bin2hex(random_bytes(16))` always 32 characters?